books/models.py

```diff
@@ -211,6 +211,8 @@ class Narration(models.Model):
                                 choices=Language.choices,
                                 blank=False)
 
+    duration_sec = models.DurationField(_('Duration'), blank=True, null=True)
```

Comment:
Let's fix a small issue, while we are here. Use `duration` rather than `duration_sec`. Using `duration_sec` made sense if we used an integer fiield. Here we are using a proper DurationField type and it doesn't look like number of seconds: https://github.com/belaudiobooks/data/blob/main/data.json#L6275